src/tracer/lightstep_tracer_impl.cpp

```diff
@@ -120,7 +120,15 @@ LightStepTracerImpl::Extract(
 // Flush
 //------------------------------------------------------------------------------
 bool LightStepTracerImpl::Flush() noexcept {
-  return recorder_->FlushWithTimeout(std::chrono::hours(24));
+  return recorder_->FlushWithTimeout(std::chrono::seconds(10));
```

Comment:
please lift the 10s to a constant somewhere more apparent?

Reply:
done